### PR TITLE
upd(an-anime-game-launcher-bin): `2.1.5` -> `2.2.0`

### PR DESCRIPTION
**Pull request by Mar0xy**
(libappdindicator3-1 is required otherwise it won't work so I don't know how you would go about when it comes to debian)

### CODE REVIEW

**Comment by wizard-28** [comment]
@Henryws doesn't work in Debian, so... what to do?

File: packages/an-anime-game-launcher-bin/an-anime-game-launcher-bin.pacscript [hunk 1]
```diff
@@ -1,9 +1,9 @@
 name=an-anime-game-launcher-bin
```
Comment (wizard-28) [inline packages/an-anime-game-launcher-bin/an-anime-game-launcher-bin.pacscript:1]:
```suggestion
# Detect if the OS is Debian and error out
if [[ -f /etc/debian_version ]]; then
    fancy_message error "This script doesn't work on Debian"
    cleanup # Trigger pacstall cleanups
    exit 1
fi

name=an-anime-game-launcher-bin
```

Error out if the OS is detected as Debian.

We'll add a variable (or an array) in the future where you can specify what OSes the package doesn't work on. But for now you'll have to manually program it and error out.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by Mar0xy** [comment]
Seems like that didn't work as it just stopped at "sourcing pacscript..." for all tests.

File: packages/an-anime-game-launcher-bin/an-anime-game-launcher-bin.pacscript (hunk 1 above)
Comment (Henryws) [inline packages/an-anime-game-launcher-bin/an-anime-game-launcher-bin.pacscript:1]:
```suggestion
if [[ "$(lsb_release -si)" == "Debian" ]]; then
    fancy_message error "This script doesn't work on Debian"
    cleanup # Trigger pacstall cleanups
    exit 1
fi
name=an-anime-game-launcher-bin
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Mar0xy** [comment]
That one seems to just be ignored due to lsb_release missing "Line 1: lsb_release: command not found"

File: packages/an-anime-game-launcher-bin/an-anime-game-launcher-bin.pacscript [hunk 2]
```diff
@@ -1,9 +1,14 @@
+if [[ "$(lsb_release -si)" == "Debian" ]]; then
+    fancy_message error "This script doesn't work on Debian"
+    cleanup # Trigger pacstall cleanups
+    exit 1
+fi
```
Comment (Henryws) [inline packages/an-anime-game-launcher-bin/an-anime-game-launcher-bin.pacscript:5]:
```suggestion
if [[ "$(grep -Po 'ID=\K[^"]+' /etc/os-release)" == "debian" ]]; then
    fancy_message error "This script doesn't work on Debian"
    cleanup # Trigger pacstall cleanups
    exit 1
fi
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by wizard-28** [comment]
Linking this with https://github.com/pacstall/pacstall/pull/510, as the error messages aren't getting printed on Debian systems

**Comment by wizard-28** [comment]
> Linking this with [pacstall/pacstall#510](https://github.com/pacstall/pacstall/pull/510), as the error messages aren't getting printed on Debian systems

This has been merged in `develop`, and will be released soon, merging this PR meanwhile.

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**